src/main/java/nl/tudelft/contextproject/script/Script.java

```diff
@@ -123,6 +135,13 @@ private void initPresetLoading() {
             t.initPreset();
         }
     }
+    
+    /**
+     * Init the timer for calling updateOldCam() with a 1 second delay.
+     */
+    private void initTimer() {
```

Comment:
Believe it or not, there is such a thing as putting too little into methods. It is clearer if you just put `timer = new Timer()` in the constructor of Script, seeing as this `initTimer()` method is only called there.